core/src/main/java/org/apache/accumulo/core/iterators/OrIterator.java

```diff
@@ -30,36 +33,66 @@
 import org.apache.accumulo.core.data.Key;
 import org.apache.accumulo.core.data.Range;
 import org.apache.accumulo.core.data.Value;
+import org.apache.commons.lang.StringUtils;
 import org.apache.hadoop.io.Text;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * An iterator that handles "OR" query constructs on the server side. This code has been adapted/merged from Heap and Multi Iterators.
```

Comment:
Yeah, that's a good point since the big-picture was to make this "user-facing" which HeapIterator and MultiIterator are not (they're actually "system" iterators, lol). That comment likely predates the package separation.